docs/plugins/zoom.md

```diff
@@ -0,0 +1,70 @@
+# Zoom Integration
+
+!!! warning
+    This feature is only available in servers where the
+    [Zoom Plugin](https://github.com/indico/indico-plugins/tree/master/vc_zoom) is installed.
+
+Indico allows meeting organisers to create a Zoom meeting and manage it directly from the management area of their
+event. Once created, the connection information of the Zoom meeting is displayed to the participants of the meeting,
+and additional buttons are added to the interface letting participants to easily connect to the Zoom meeting, and
+organisers to connect their physical meeting room to the Zoom meeting.
+
+## Managing Zoom Meetings
+
+On the management page of your Indico event click on _Videoconference_ under the tab _Services_ on the left banner.
+
+![](../assets/zoom/videoconf_menu.png)
+
+Here you can _create_ a new Zoom meeting or _Add_ an existing one to your event.
+
+![](../assets/zoom/videoconf_list.png)
+
+Simply click on one option then select the Zoom videoconference service.
```

Comment:
Only when there are multiple services available; otherwise this step doesn't happen. Worth mentioning this?

Reply:
Probably not, if there's only one, they'll see the other dialog straight away. We can perhaps remove this in the CERN version of things once Vidyo goes  away.